moviepy/__init__.py

```diff
@@ -1 +1,57 @@
+"""
+Imports everything that you need from the MoviePy submodules so that every thing
+can be directly imported like `from moviepy import VideoFileClip`.
+
+In particular it loads all effects from the video.fx and audio.fx folders
+and turns them into VideoClip and AudioClip methods, so that instead of
+``clip.fx(vfx.resize, 2)`` or ``vfx.resize(clip, 2)``
+you can write ``clip.resize(2)``.
+
+"""
+
+import inspect
+
 from .version import __version__
+
+
+# Clips
+from .video.io.VideoFileClip import VideoFileClip
+from .video.io.ImageSequenceClip import ImageSequenceClip
+from .video.io.downloader import download_webfile
+from .video.VideoClip import VideoClip, ImageClip, ColorClip, TextClip, BitmapClip
+from .video.compositing.CompositeVideoClip import CompositeVideoClip, clips_array
+from .video.compositing.concatenate import concatenate_videoclips
+
+from .audio.AudioClip import AudioClip, CompositeAudioClip, concatenate_audioclips
+from .audio.io.AudioFileClip import AudioFileClip
+
+# FX
+from .video import fx as vfx
+import moviepy.audio.fx as afx
+import moviepy.video.compositing.transitions as transfx
+
+# Tools
+import moviepy.video.tools as videotools
+import moviepy.video.io.ffmpeg_tools as ffmpeg_tools
+from .tools import convert_to_seconds
```

Comment:
Some imports are relative (`from .foo.bar`) and others absolute (`import moviepy.foo.bar`). Why not use absolute imports for all?

Reply:
That's just the way it was in editor.py. Thanks for noticing, I'll change it